app/src/main/java/com/example/africanid/MainActivity.java

```diff
@@ -3,12 +3,64 @@
 import androidx.appcompat.app.AppCompatActivity;
 
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        Button button = findViewById(R.id.buttonS);
```

Comment:
The name for button would be better if you make more descriptive, what do you think about ?
buttonS => buttonSubmit or btnSubmit